### ISSUE TITLE
ZSH integration throws an error

### ISSUE THREAD

**Issue by gowthamsadasivam** [sedwards2009/extraterm#61]
I just followed the instructions from the Docs. I'm on Ubuntu 14.04.3 with ZSH as default shell. I downloaded the file `extraterm-commands-0.28.0.zip` and extracted the scripts to a directory and tried sourcing it inside my `~/.zshrc` file by adding the line `source /home/xxxx/.extraterm/extraterm-commands/setup_extraterm_zsh.zsh`. Now when I opened `Extraterm` I saw this message,

`/home/xxxx/.extraterm/extraterm-commands/setup_extraterm_zsh.zsh:export:16: not valid in this context: %}%m%#`

How to fix this?

**Comment by sedwards2009** [comment]
Are you able to even start up Extraterm? I'm trying to reproduce your problem using an Ubuntu 14.04 virtual machine but I don't have a recent enough version of glibcxx.

**Comment by sedwards2009** [comment]
Can you tell me which version of zsh you are using?

**Comment by gowthamsadasivam** [comment]
@sedwards2009 Thanks for the efforts and reply.

My ZSH version is : `zsh 5.0.2 (x86_64-pc-linux-gnu)`

And yes, I also had the same issue with **glibcxx** then I installed and upgraded the version using,

`sudo add-apt-repository ppa:ubuntu-toolchain-r/test `
`sudo apt-get update`
`sudo apt-get upgrade`

I just upgraded all the **glibc** libraries available that needed to run `Extraterm`. But now stuck with `ZSH` error.

Thanks.
